### PR TITLE
Api provides functionality for subscribers to comment on articles

### PR DESCRIPTION
**Pull request by LucasKnudsen**
[**PT-Story:**  ](https://www.pivotaltracker.com/story/show/178175967)
### User Story 
``` 
As an API
In order to serve my paying consumers with the ability to express their opinion
I want subscribers to be able to comment on articles
``` 
## Changes proposed in this pull request: 
* Opens create route for Comments
* Restricts create action to Users

## What I have learned working on this feature: 
* A bit of seeding magic
* Using nested routing

### CODE REVIEW

File: spec/requests/api/subscriber/subscribers_can_write_comments_spec.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+    it 'is expected to create an instance of a Comment with the expected body content' do
+      expect(Comment.last['body']).to eq 'What a great website, I have subscribed solely based on the awesome design!'
+    end
+  end
+
+  describe 'Unsuccessfully as a visitor' do
+    before do
+      post "/api/articles/#{article.id}/comments", params: {
+        body: 'ITS FAKE NEWS'
```
Comment (arunbhalli) [inline spec/requests/api/subscriber/subscribers_can_write_comments_spec.rb:34]: is it possible to write like this ('IT/'S FAKE NEWS') here at the place of 'ITS FAKE NEWS'?

Comment (LucasKnudsen) [reply spec/requests/api/subscriber/subscribers_can_write_comments_spec.rb:34]: It is, but these South Californian people ain't so good with spelling :3

**Review by christerforsgren91 — APPROVED** [review]
🏎️

File: spec/requests/api/subscriber/subscribers_can_write_comments_spec.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    it 'is expected to respond with a success message' do
+      expect(response_json['message']).to eq 'Your comment has been published'
+    end
+
+    it 'is expected to add a comment to the article' do
+      expect(article.comments.count).to eq 1
+    end
+
+    it 'is expected to create an instance of a Comment with the expected body content' do
+      expect(Comment.last['body']).to eq 'What a great website, I have subscribed solely based on the awesome design!'
```
Comment (1CIM) [inline spec/requests/api/subscriber/subscribers_can_write_comments_spec.rb:27]: NO CAP

**Review by 1CIM — APPROVED** [review]

File: spec/requests/api/subscriber/subscribers_can_write_comments_spec.rb (hunk 2 above)
Comment (LucasKnudsen) [reply spec/requests/api/subscriber/subscribers_can_write_comments_spec.rb:27]: I see

**Review by arunbhalli — COMMENTED** [review]
good job

**Pull request merged**